From dcbf163d44a2814928a4e59076c1f32f58ca9dd2 Mon Sep 17 00:00:00 2001
From: Stephen Foulds <stephen.foulds@consult.red>
Date: Fri, 27 Nov 2020 14:44:33 +0000
Subject: [PATCH] Change config file dir from etc to opt
Upstream-Status: Pending
Source: Pending

Reason for change: Dobby needs to read/write to the smcroute
config file. By default, smcroute reads the config from /etc
which is a read-only directory on Xi6. Change to use /opt
instead. File does not need to exist at smcroute startup

Test procedure: Create /opt/smcroute.conf file and add a rule.
Restart smcroute. The new rule should be detected.

Signed-off-by: Stephen Foulds <stephen.foulds@consult.red>
---
 smcroute.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/smcroute.service.in b/smcroute.service.in
index 910f486..8fcda04 100644
--- a/smcroute.service.in
+++ b/smcroute.service.in
@@ -9,7 +9,7 @@ Requires=network-online.target
 
 [Service]
 Type=simple
-ExecStart=@SBINDIR@/smcrouted -n -s
+ExecStart=@SBINDIR@/smcrouted -n -s -f /opt/smcroute.conf
 
 # Hardening settings
 NoNewPrivileges=true
-- 
2.7.4

